care/emr/api/viewsets/inventory/dispense_order.py

```
@@ -69,17 +69,15 @@ def authorize_pharmacist(self, facility):
     def authorize_create(self, instance):
         facility = self.get_facility_obj()
         if self.authorize_pharmacist(facility):
             return
         if self.authorize_location_write(
-            get_object_or_404(
-                FacilityLocation, external_id=instance.location, facility=facility
-            )
+            get_object_or_404(FacilityLocation, external_id=instance.location)
         ):
             return
         raise PermissionDenied("You do not have permission to create dispense order")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)